db/migrations/0002_user_order_ticket_movie_db_movie_title_5d0841_idx_and_more.py

```diff
@@ -0,0 +1,91 @@
+# Generated by Django 4.0.2 on 2023-09-04 21:47
+
+from django.conf import settings
+import django.contrib.auth.models
+import django.contrib.auth.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('auth', '0012_alter_user_first_name_max_length'),
```

Comment:
change ' to ", use Black

Reply:
This file is created automatically, should I really change every automatically created file?

Reply:
yes, please, it is best practies, `Black` do it automatically

Reply:
Okay, all done. Thank you!